.travis.yml

```diff
@@ -8,7 +8,15 @@ services:
   - postgresql
 install:
   - "pip install -e ."
-  - "pip install -r requirements-dev.txt
+  - "pip install -r requirements-dev.txt --use-wheel"
```

Comment:
The main problem was the missing closing quote which wasn't valid, so I just pasted a working line from another project.